usb/esp_modem_usb_dte/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 idf_component_register(SRCS "esp_modem_usb.cpp" "esp_modem_usb_api_target.cpp" "esp_modem_usb_c_api.cpp"
                        REQUIRED_IDF_TARGETS esp32s2 esp32s3
                        PRIV_INCLUDE_DIRS "private_include"
-                       INCLUDE_DIRS "include")
+                       INCLUDE_DIRS "include"
+                       PRIV_REQUIRES usb usb_host_cdc_acm esp_modem driver esp_netif )
```

Comment:
usb_host_cdc_acm and esp_modem components dependencies are defined in idf_component.yml and we try to not redefine them here, could you please remove these 2 components from the list?

Reply:
`usb` requirement is propagated from usb_host_cdc_acm.
`driver` and `esp_netif` are propagated from esp_modem.

So there should be no reason for change